### PR TITLE
Build: Release PR v3.1.25

### PR DESCRIPTION
**Pull request by vrundakansara**
### Description
- Final release PR. Please check all the checklist below before merging.
---
### Checklist:
- [ ] Finalize readme.txt
- [ ] Add doc links in the readme.txt
- [ ] Check if Gutenberg Blocks/Astra Notices library is updated to latest - Run `composer update`
- [ ] Run to update packages        : `npm audit fix`
- [ ] Run to produce rtl files : `grunt rtl`
- [ ] Run to update the Template Library    : `wp astra-sites sync --force`
- [ ] PHPCS - vendor/bin/phpcs
- [ ] Run to update version number  : `grunt version-bump --ver=<version-number>`
- [ ] Update - Tested Upto - readme.txt
- [ ] Update readme.txt file and then run: `grunt readme`
- [ ] Generate POT file             : `grunt i18n`
- [ ] Add draft Release notes on github
- [ ] Add draft changelogs to wpastra.com
- [ ] Open PR to backport branch to `next-release`
---

v3.1.25 - 20-February-2023
- Improvement: Hardened the security.

### CODE REVIEW

**Comment by vrundakansara** [comment]
> Please update the dependency in the lock file. Also, Can we create a pre-release branch first and update the checklists like this PR: #508

@sadanandl - Adding the label to the same PR can work as well. All the pre-release changes are already executed here as well. And the lock files are also updated already.

File: inc/importers/batch-processing/class-astra-sites-batch-processing-gutenberg.php [hunk 1]
```diff
@@ -145,8 +145,8 @@ public function import_single_post( $post_id = 0 ) {
 			if ( ! empty( $ids_mapping ) ) {
 				// Replace ID's.
 				foreach ( $ids_mapping as $old_id => $new_id ) {
-					$content = str_replace( '[wpforms id="' . $old_id, '[wpforms id="' . $new_id, $content );
-					$content = str_replace( '{"formId":"' . $old_id . '"}', '{"formId":"' . $new_id . '"}', $content );
+					$content = str_replace( '[wpforms id=\"' . $old_id, '[wpforms id=\"' . $new_id, $content );
```
Comment (sagarb90) [inline inc/importers/batch-processing/class-astra-sites-batch-processing-gutenberg.php:148]: Have we intentionally changed this?

Comment (vrundakansara) [reply inc/importers/batch-processing/class-astra-sites-batch-processing-gutenberg.php:148]: Yes, we have @sagarb90 . The WP Forms import is broken in master branch as well.

**Comment by sadanandl** [comment]
> > Please update the dependency in the lock file. Also, Can we create a pre-release branch first and update the checklists like this PR: #508
> 
> @sadanandl - Adding the label to the same PR can work as well. All the pre-release changes are already executed here as well. And the lock files are also updated already.

I think we need to update the dependency in the lock file so that maybe the phpcs workflow will also get passed.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -1,23 +1,67 @@
...
+
+    - name: Set composer token
+      run: composer config github-oauth.github.com ${{ secrets.PRIVATE_ACCESS_TOKEN }}
+
+    - name: Composer Install
+      uses: ramsey/composer-install@v1
+
+    - name: NPM Dependencies
+      if: always()
+      run: yarn install
```
Comment (sadanandl) [inline .github/workflows/ci.yml:45]: Why we are installing yarn here? any reason?

Comment (vrundakansara) [reply .github/workflows/ci.yml:45]: It is used on line 57 - `yarn run lint:js:fix`

Comment (sadanandl) [reply .github/workflows/ci.yml:45]: Yes, that's the line we have added for the lint fix. You can see the workflow failed. Can we check it?

Also one note: on premium repo we are not using yarn.
https://github.com/brainstormforce/astra-pro-sites/pull/915/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR32-R51

**Review by sagarb90 — APPROVED** [review]

**Review by sadanandl — APPROVED** [review]

**Pull request merged**